misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -173,20 +174,25 @@ def _get_issue_presentation(self) -> str:
         return issue_presentation
 
     def to_text(self) -> str:
-        return f"{self.error_type} {self.issue_title} ({self._get_issue_presentation()}) in {self.location}: {crop_text(self.error_message)}"
+        result = f"{self.error_type} {self.issue_title} ({self._get_issue_presentation()}) in {self.location}: {crop_text(self.error_message)}"
+        if self.error_details is not None:
+            result += f"\n{crop_text(self.error_details)}"
+        return result
 
     def to_markdown(self) -> str:
         if self.location_url is None:
             location_markdown = self.location
         else:
             location_markdown = f'<a href="{self.location_url}">{self.location}</a>'
 
-        return f'{self.error_type} <a href="{self.issue_url}">{self.issue_title} ({self._get_issue_presentation()})</a> in {location_markdown}:\n{format_error_message(self.error_message)}'
+        result = f'{self.error_type} <a href="{self.issue_url}">{self.issue_title} ({self._get_issue_presentation()})</a> in {location_markdown}:\n{format_error_message(self.error_message)}'
+        if self.error_details is not None:
+            result += f"\n{format_error_message(self.error_details)}"
+        return result
 
 
 @dataclass(kw_only=True, unsafe_hash=True)
 class ObservedErrorWithLocation(ObservedError):
```

Comment:
We might want to pull up the max-length fields and use them there.

Reply:
Wondering if we still need this class... I see that `location` was pulled up.

Reply:
Done. I think so, it's output is still different.

Reply:
Ah, I mistakenly thought `ObservedErrorWithLocation` inherits `ObservedErrorWithIssue` in which case there would have been not much diff left. All good!